dotcom-rendering/src/lib/braze/buildBrazeMessaging.ts

```diff
@@ -108,6 +108,7 @@ export const buildBrazeMessaging = async (
 		const sdkLoadTimeTaken = endPerformanceMeasure();
 		const ophan = await getOphan(renderingTarget);
 		ophan.record({
+			// @ts-expect-error -- is this event dropped by Ophan?
```

Comment:
Maybe adjust these comments to state that these events are being dropped and add a link to this PR for further context or a note that other teams will remove it later on.